src/components/Modal.jsx

```diff
@@ -46,13 +47,50 @@ class Modal extends React.Component {
       if (e.constructor.name === 'CustomEvent' && e.target.id === `${this.props.id}-hotspot`) {
         this.toggleVisibility(true);
       } else if ((Date.now() - this.state.lastActivation) > 100 &&
-                 (!AFRAME.utils.device.isMobile())) {
+        (!AFRAME.utils.device.isMobile())) {
         this.toggleVisibility(false);
       }
     });
   }
 
   /**
+   * Determines if a sound is set for this modal.
+   *
+   * @returns {boolean}
+   *   Returns true if a sound is set, otherwise returns false.
+   */
+  hasSound() {
```

Comment:
I was totes wrong on this, you defaults are type checked. :stop_sign: This is good to go!